src/reset-password/service.ts

```diff
@@ -0,0 +1,20 @@
+import nodemailer from 'nodemailer';
+import { User } from '../user/types';
+
+
+export async function sendResetPasswordEmail(user: User) {
+
+    // create reusable transporter object using the default SMTP transport
+    const transporter = nodemailer.createTransport(process.env.SMTP_URL);
+
+    // send mail with defined transport object
+    const info = await transporter.sendMail({
+        from: '"John Doe" <system@pegasi.badgateway.net>', // sender address
+        to: user.identity.substring(7), // list of receivers
+        subject: 'Reset Password Request', // Subject line
+        text: 'Please click below link to reset your password!', // plain text body
```

Comment:
This can be done in a future PR, but the emails might need to be improved a bit =)